python/tank/pipelineconfig_factory.py

```diff
@@ -15,22 +15,46 @@
 import cPickle as pickle
 
 from .errors import TankError
-from .platform import constants 
+from .platform import constants
 from .util import shotgun
 from . import pipelineconfig_utils
 from .pipelineconfig import PipelineConfiguration
 
 
+def _massage_path(path):
```

Comment:
I don't like this method name. You mention 'normalize' twice in the docstring - I suggest naming the method `_normalize_path()`